src/python/national_voter_file/transformers/README.md

```diff
@@ -1,33 +1,37 @@
 # Adding a new state
 
-Here are the basic steps you want to perform:
+* Create a new transformer from an existing one:
+
+```cp -R us_states/mi us_states/{new_state_postal_initials}```
+
+* Update it accordingly. Besides updating the methods, you'll need to update these variables:
 
-* copy `state_transformer_template.py` to `us_states/{state_postal_initials}/transfomer.py` (e.g. `us_states/pa/transformer.py`)
-* Update the values in the headers
   1. `state_path` - this is the two letter state code (e.g. tx)
   2. `state_name` - CamelCase state namme (e.g. NewYork)
   3. `sep` - Delimiter between columns
   4. `date_format` - The python date parser format string for how to read dates for this voter file.
-  5. `input_fields` - a list of column names for the source file. This should be set to `None` when the file has a header row.
+  5. `col_map` - A mapping of column names that don't need to transformed in any way.
```

Comment:
Small typo, guessing this should be "don't need to be transformed in any way". Also, might add that it also maps columns that are empty/None